rllib/utils/compression.py

```diff
@@ -27,7 +27,7 @@ def compression_supported():
 @DeveloperAPI
 def pack(data):
     if LZ4_ENABLED:
-        data = pyarrow.serialize(data).to_buffer().to_pybytes()
+        data = pickle.dumps(data)
```

Comment:
@ericl is this OK for rllib? pickle5 mode should be as fast as pyarrow or even faster

Reply:
The performance test results are as follows:
Before (pyarrow):
```
Compression speed: 1205.8624 MB/s
Compression ratio: 87.43
Decompression speed: 1418.8544 MB/s
```

After (pickle5):
```
Compression speed: 1074.7904 MB/s
Compression ratio: 88.6
Decompression speed: 1019.0848 MB/s
```